pep_sphinx_extensions/pep_zero_generator/parser.py

```diff
@@ -140,6 +140,7 @@ def details(self) -> dict[str, str | int]:
             "shorthand": self.shorthand,
             # the author list as a comma-separated with only last names
             "authors": ", ".join(author.full_name for author in self.authors),
+            "python_version": self.python_version or "",
```

Comment:
```suggestion
            # The targeted Python-Version (if present) or the empty string
            "python_version": self.python_version or "",
```

Reply:
The comment on line 139 is useful, but this one duplicates the code and the one on line 141 is not accurate anymore 🙂